Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs

```diff
@@ -19,6 +19,8 @@ public enum LeverDirection
             x, y, z
         }
 
+        [Tooltip("An optional game object to which the button will be connected. If the game object moves the button will follow along.")]
```

Comment:
Whoops, that's why you shouldn't copy/paste code haha. I will amend and push again today, thanks.